shared/menubar/remote-proxy.desktop.js

```diff
@@ -67,6 +69,8 @@ const mergeProps = stateProps => ({
   loggedIn: stateProps.loggedIn,
   username: stateProps.username,
   conversations: stateProps.conversations,
+  broken: stateProps.broken,
```

Comment:
broken x 2?

Reply:
WDYM? This is `mergeProps`, where the previous one is in `mapStateToProps`.

Reply:
oh github made this look like the same object